_includes/release-notes/51new-features.md

```diff
@@ -0,0 +1,237 @@
+| `os.isiOS`             | Boolean | Returns `true` if the user's operating system is iOS.                |
+| `os.isAndroid`         | Boolean | Returns `true` if the user's operating system is Android.            |
+| `os.isOSX`             | Boolean | Returns `true` if the user's operating system is Mac OS X.           |
+| `os.isLinux`           | Boolean | Returns `true` if the user's operating system is Linux.              |
+| `os.isSolaris`         | Boolean | Returns `true` if the user's operating system is Solaris.            |
+| `os.isFreeBSD`         | Boolean | Returns `true` if the user's operating system is FreeBSD.            |
+| `deviceType.isDesktop` | Boolean | Returns `true` if the user's device is a desktop device.             |
+| `deviceType.isiPad`    | Boolean | Returns `true` if the user's device is an iPad.                      |
+| `deviceType.isiPhone`  | Boolean | Returns `true` if the user's device is an iPhone.                    |
+| `deviceType.isPhone`   | Boolean | Returns `true` if the user's device is a phone.                      |
+| `deviceType.isTablet`  | Boolean | Returns `true` if the user's device is a tablet.                     |
+| `deviceType.isTouch`   | Boolean | Returns `true` if the user's device is a touch device.               |
+| `deviceType.isWebView` | Boolean | Returns `true` if the user's device is a WebView device.             |
+
+For a list of deprecated `Env` APIs, see: [Deprecated API Properties - `tinymce.Env`](#deprecatedapiproperties-tinymceenv).
+
+### Added new `referrer_policy` setting
+Used for setting the level of referrer information sent when loading plugins and CSS. Referrer policies can be used to:
+
+* Improve the privacy of end-users.
+* Assist with server-side filtering of cross-origin requests for {{site.productname}} resources.
+
+For information on using the `referrer_policy` setting, see: [Integration and setup options - `referrer_policy`]({{site.baseurl}}/configure/integration-and-setup/#referrer_policy).
+
+### Added a dark content css skin
+A dark CSS definition for `content_css` has been added to compliment the dark user interface skin.
+
+For example:
+
+{% include codepen.html id="dark-mode" %}
+
+For information on using the dark version of the default skin, see: [Customizing the editor UI - Skins]({{site.baseurl}}/general-configuration-guide/customize-ui/#skins).
+
+### Added border width to Table cell dialog
+The table plugin has been updated to include a **Border width** field in the **Cell Properties** dialog. The field will accept any [valid CSS units](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units#Lengths).
+
+For example:
+
+<img alt="Cell Properties Dialog with new Border Width field" src="{{site.baseurl}}/images/border-width-cell-props.png" style="width:380px;border: 1px solid #036DD5" />
+
+### Changed the default `toolbar_drawer` to `floating`
```

Comment:
Should we include how to change this back to the 5.0 behaviour? eg:

```
tinymce.init({
  ...
  toolbar_drawer: false
});
```

Reply:
Added